app/components/forms.tsx

```diff
@@ -80,9 +82,11 @@ export function TextareaField({
 				aria-describedby={errorId}
 				{...textareaProps}
 			/>
-			<div className="px-4 pb-3 pt-1">
-				{errorId ? <ErrorList id={errorId} errors={errors} /> : null}
-			</div>
+			{errorId ? (
+				<div className="px-4 pb-3 pt-1">
+					<ErrorList id={errorId} errors={errors} />
+				</div>
+				) : null}
```

Comment:
Could you show how this impacts the UI? I'm worried that this will result in jerkiness when error messages appear. I believe that's why we have the divs always rendered.

Reply:
Sure — by always rendering, the empty div still took up some space which could cause some weird effects when working with displays types like grid and flex

Lmk if you'd like me to add some supporting screenshots as well